pkg/container-disk/container-disk_test.go

```diff
@@ -182,8 +191,11 @@ var _ = Describe("ContainerDisk", func() {
 					},
 				})
 
-				vmi := api.NewMinimalVMI("fake-vmi")
-				appendContainerDisk(vmi, "r0")
+				vmi := libvmi.New(
+					libvmi.WithContainerDisk("r0", someImage),
```

Comment:
you can use `WithResourceMemory`, `WithResourceCPU`, `WithLimitMemory`, `WithLimitCPU` and drop following resource and limits assignment